BREAKING.md

```diff
@@ -15,6 +15,26 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 -   Avoid using code formatting in the title (it's fine to use in the body).
 -   To explain the benefit of your change, use the [What's New](https://fluidframework.com/docs/updates/v1.0.0/) section on FluidFramework.com.
 
+## 2.0.0-internal.4.1.0 Breaking changes
```

Comment:
```suggestion
## 2.0.0-internal.5.0.0 Breaking changes
```

Reply:
This is going into the `main` branch behind a runtime option (default off), but I wanted to make a note of in in the `BREAKING.md` file